src/test/java/org/openrewrite/java/logging/jboss/LoggerLevelArgumentToMethodTest.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2025 the original author or authors.
+ * <p>
+ * Licensed under the Moderne Source Available License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://docs.moderne.io/licensing/moderne-source-available-license
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.logging.jboss;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+class LoggerLevelArgumentToMethodTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.recipe(new LoggerLevelArgumentToMethod());
+    }
+
+    @DocumentExample
+    @ParameterizedTest
+    @ValueSource(strings = {"trace", "debug", "info", "warn", "error", "fatal"})
+    void replaceLevelArguments(String level) {
+        rewriteRun(
+          //language=java
+          java(
+            """
+              import org.jboss.logging.Logger;
+
+              import static org.jboss.logging.Logger.Level.%1$S;
+
+              class Test {
+                  void test(Logger logger, Object msg, Throwable t, String fqcn) {
+                      logger.log(Logger.Level.%1$S, msg);
```

Comment:
Any suggestions on how to avoid the syntax errors reported by IDEA ("Identifier expected" in this case) other than removing the `//language=java` hint?

Reply:
No, that's what it is... Just ignore this error (hey, you are smarter than your IDE, so that's a good thing, right 😉).